src/System.Runtime.Serialization.Primitives/tests/System/Runtime/Serialization/CollectionDataContractAttributeTests.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
```// The .NET Foundation licenses this file to you under the MIT license.```
is missing from the license. please add it.

Reply:
Fixed for all cases